.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: bazel
+    directory: /
+    schedule:
+      interval: weekly
+  - package-ecosystem: cargo
+    directories:
+      - /
+      - examples/grpc_auth_random
+      - examples/hello_world
+      - examples/http_auth_random
+      - examples/http_headers
```

Comment:
It's missing the other 4 examples.

Reply:
I didn't found any dependencies except the local one, that's why I didn't provided them but I can add them of course